docs/.vuepress/plugins/chart-editor/components/ChartEditor.vue

```diff
@@ -13,15 +13,18 @@
 </template>
 
 <script>
-import {Chart} from 'chart.js';
+import {Chart, registerables} from 'chart.js';
+import {merge} from 'chart.js/helpers';
 import * as Utils from '../utils';
 
 // Components
 import ChartActions from './ChartActions.vue';
 import ChartView from './ChartView.vue';
 import CodeEditor from './CodeEditor.vue';
 
-const CHART_DEFAULTS = Chart.helpers.merge({}, Chart.defaults);
+Chart.register(...registerables);
+
+const CHART_DEFAULTS = merge({}, Chart.defaults);
```

Comment:
I suspect `Chart.defaults` to not contain the plugin defaults yet (which is registered in `ChartView`), thus the broken examples (thought, I'm not sure why it works before). We could try to move the plugin registration here, right after line 25. Could also be related to the proxied options, but I'm not sure if defaults are proxied.